app/views/admin/includes/_head.html.erb

```diff
@@ -1,16 +1,19 @@
-<title>
-  <%= "#{@page_title.titlecase} - " if @page_title %>ShinyCMS
-</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
 
-<%= csrf_meta_tags %>
-<%= csp_meta_tag   %>
+    <title>
+      <%= "#{@page_title.titlecase} - " if @page_title %>ShinyCMS
+    </title>
 
-<%= favicon_link_tag %>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag   %>
 
-<%= stylesheet_link_tag 'https://fonts.googleapis.com/css?family=Poppins&display=swap' %>
-<%= stylesheet_link_tag 'shinycms/admin_area', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= favicon_link_tag %>
 
-<%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
-<%= javascript_pack_tag 'admin_area',  'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag 'https://unpkg.com/@coreui/coreui@3.0.0-beta.1/dist/css/coreui.min.css' %>
+    <%= stylesheet_link_tag 'https://fonts.googleapis.com/css?family=Poppins&display=swap'          %>
+    <%= stylesheet_link_tag 'shinycms/admin_area', media: 'all', 'data-turbolinks-track': 'reload'  %>
 
-<%= javascript_include_tag Ckeditor.cdn_url %>
+    <% if action_name == 'edit' || controller_name == 'shared_content' %>
```

Comment:
For clarity, consider extracting a method that reveals the intent of this condition, while also hiding the implementation details behind it, e.g.

```erb
<%= javascript_include_tag Ckeditor.cdn_url if editable? %>
```

```rb
def editable?
  action_name == 'edit' || ...
end
```

The extracted method would go in one of the `app/helpers/` files.

Reply:
Ooh, that's satisfyingly neat :)